docs/src/customization.md

```diff
@@ -4,42 +4,290 @@ description: Customize your drop-down menu and instant search results page.
 layout: page.html
 ---
 
-## Introduction
+<div class="alert alert-warning">**Reminder**: You should never directly edit the plugin files because all your changes will be lost at every plugin update.</div>
 
-### Autocomplete
+## Autocomplete
+
+### Introduction
 
 *The autocomplete experience can be enabled in the **Autocomplete** section of the plugin.*
 
 The autocomplete dropdown menu is powered by **autocomplete.js**, a JavaScript library that eases the process of building rich dropdown menus.
 
 Please checkout the [github repository of the autocomplete.js library](https://github.com/algolia/autocomplete.js/).
 
-### Instant Search Results Page
+<div class="alert alert-info">Whatever changes you make, be sure to understand how the autocomplete.js library work by checking out their official documentations.</div>
+
+### Customization
+
+To start customizing your dropdown menu, copy/paste the `wp-content/plugins/algolia/templates` directory in your own theme, and rename it to `algolia`. If your theme is named `mytheme`, then the folder should look like: `wp-content/themes/mytheme/algolia`.
+
+You can then edit the `algolia/autocomplete.php` file to customize the autocomplete dropdown menu (suggestions, header, footer) templates & associated JavaScript code.
+
+<div class="alert alert-info">All autocomplete.js templates are using the underscore templating, learn more about it reading this [complete documentation](http://www.2ality.com/2012/06/underscore-templates.html).</div>
+
+#### Edit the suggestion templates
+
+Each autocomplete.js source will display its suggestions with an associated template. To configure them, edit the `algolia/autocomplete.php` file and locate the `tmpl-autocomplete-*-suggestion` templates:
+
+#### Post template
+
+This is the template used for all **Post**-based types.
+
+```html
+<script type="text/html" id="tmpl-autocomplete-post-suggestion">
+  <a class="suggestion-link" href="{{ data.permalink }}" title="{{ data.post_title }}">
+    <# if ( data.thumbnail_url ) { #>
+    <img class="suggestion-post-thumbnail" src="{{ data.thumbnail_url }}" alt="{{ data.post_title }}">
+    <# } #>
+    <div class="suggestion-post-attributes">
+      <span class="suggestion-post-title">{{{ data._highlightResult.post_title.value }}}</span>
+
+      <#
```

Comment:
Do you think we could put this elsewhere, kinda ugly. Maybe We can add an attribute to the records on the fly when fetching results from Algolia.